src/main/java/xyz/nucleoid/map_templates/BlockBounds.java

```diff
@@ -50,7 +50,7 @@ public static BlockBounds ofChunk(Chunk chunk) {
     public static BlockBounds ofChunk(ChunkPos chunk, HeightLimitView world) {
         return new BlockBounds(
                 new BlockPos(chunk.getStartX(), world.getBottomY(), chunk.getStartZ()),
-                new BlockPos(chunk.getEndX(), world.getTopY(), chunk.getEndZ())
+                new BlockPos(chunk.getEndX(), world.getTopYInclusive(), chunk.getEndZ())
```

Comment:
Technically this returns different value than just the old getTopY (it's one block lower), through this shouldn't cause any issues